dipy/reconst/mapmri.py

```diff
@@ -159,6 +160,9 @@ def __init__(self,
             the tissue diffusivity that is used when dti_scale_estimation is
             set to False. The default is that of typical white matter
             D=0.7e-3 _[5].
+        cvxpy_solver : cvxpy solver name
```

Comment:
Adapted the documentation according to you suggestion. The default in cvxpy is simply None so it makes its own decision.